loops-and-arrays.js

```diff
@@ -12,8 +12,8 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 // Na jouw script zie je de aangepaste waardes:
 // console.log(names) geeft: ["Henkje", "Pietje", "Fredje", "Joopje"]
 
-
-
+for (let i = 0; i < names.length; i++)
+    console.log(names[i] + "je")
```

Comment:
Hey Debby. Sorry Dat het zo lang geduurt heeft! Ik was het in de drukte vergeten! Maar ik zal het nu doen voor je!